samcli/lib/test_runner/base_template.j2

```diff
@@ -0,0 +1,69 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Sample Template to deploy and run test container with Fargate
+Resources:
+
+    ContainerIAMRole:
+        Type: AWS::IAM::Role
+        Properties:
+            Description: Allows Fargate task to access S3 bucket to download tests and upload results
+            AssumeRolePolicyDocument:
+                Version: 2012-10-17
+                Statement:
+                    - Effect: Allow
+                      Principal:
+                          Service:
+                              - ecs-tasks.amazonaws.com
+                      Action:
+                          - sts:AssumeRole
+            Policies:
+                - PolicyName: ContainerPermissions
+                  PolicyDocument:
+                      Version: 2012-10-17
+                      Statement:
+                          - Sid: S3BucketAccess
+                            Effect: Allow
+                            Action:
+                                - s3:PutObject
+                                - s3:GetObject
+                            Resource: !Sub
+                                - arn:aws:s3:::${bucket}/*
+                                - bucket: !Ref S3Bucket
+                        # Generated IAM Actions #
+                          {{ generated_statements | indent(6 * 4 + 2) }}
+    TaskDefinition:
+        Type: AWS::ECS::TaskDefinition
+        Properties:
+            RequiresCompatibilities:
+                - FARGATE
+            ExecutionRoleArn: !Sub arn:aws:iam::${AWS::AccountId}:role/ecsTaskExecutionRole
+            TaskRoleArn: !Ref ContainerIAMRole
+            Cpu: 256
+            Memory: 512
+            NetworkMode: awsvpc
+            ContainerDefinitions:
+                - Name: cloud-test-python-container
+                  Image: {{ image_uri }}
+                  PortMappings:
+                      - ContainerPort: 8080
+                        Protocol: tcp
+                  LogConfiguration:
+                      LogDriver: awslogs
+                      Options:
+                          awslogs-region: !Ref AWS::Region
+                          awslogs-group: !Ref LogGroup
+                          awslogs-stream-prefix: ecs
+
+    LogGroup:
+        Type: AWS::Logs::LogGroup
+        Properties:
+            LogGroupName: cloud-test-loggroup
+
+    ECSCluster:
+        Type: AWS::ECS::Cluster
+        Properties:
+            ClusterName: cloud-test-fargate-cluster
```

Comment:
In the unlikely event a customer already has such a cluster, what should we do?

Can this value be parameterized or derived from other inputs?